UI/frontend-plugins/frontend-tools/scripts.cpp

```diff
@@ -39,14 +41,14 @@
 #define PYTHON_UI 0
 #endif
 
+#define PYTHON_VERSION "3.6"
```

Comment:
Hmm, does this have to be defined manually?

Via: <https://docs.python.org/3/c-api/apiabiversion.html> (which we use in `obs-scripting-python-import.c`)

Something like
```suggestion
#define PYTHON_VERSION PY_MAJOR_VERSION + "." + PY_MINOR_VERSION
```

However - we don't currently expose that here. I'm unsure how best to do it.